apps/v1/consts.go

```diff
@@ -0,0 +1,135 @@
+package v1
+
+const (
+	// DeploymentStatusReasonAnnotation represents the reason for deployment being in a given state
+	// Used for specifying the reason for cancellation or failure of a deployment
+	// This is set by deployer controller and consumed in retry and history commands.
```

Comment:
I am slightly afraid such comments get outdated over time

Reply:
they are for review only ;-) i will remove them before merge